cgi-bin/DW/Logic/ProfilePage.pm

```diff
@@ -851,6 +851,17 @@ sub external_services {
         };
     }
 
+    if ( my $medium = $u->prop( 'medium' ) ) {
+	my $medium = LJ::eurl( $medium );
+	push @ret, {
+	    type => 'medium',
+	    text => LJ::ehtml( $medium ),
+	    url => "https://medium.com/\@$medium",
```

Comment:
Thanks for the feedback. The URL `. . . /latest` opens the _Latest_ column of the profile. Though this was in the original request, I believe having a profile link point to a profile is the intent. Latest entrees will appear here along with any stickied or featured posts, depending on the user's preference.

If we are sticking with the specs, should I change "Medium user account" to "Medium user posts"? I ask -- so that we don't end up with conflicting information in the future.